test/integration/list/test_list.py

```diff
@@ -0,0 +1,43 @@
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+
+
+def test_list_enabled_apps(tt_cmd):
```

Comment:
Please, add the following tests:
- `instances_enabled: .` in tt env config. There are two scenarios for this:
    - . is an application. If it is not
    - . is not an app.

- `instances_enabled` points to non-existing directory.
- `instances_enabled` contains a directory, which is not an application (contains any .lua file, but not init.lua)

Reply:
added